libs/dh/market-participant/feature-market-participant/src/components/create/create.component.ts

```diff
@@ -298,15 +292,5 @@ export class DhActorsCreateActorModalComponent extends WattTypedModal {
 
       this.close(true);
     }
-
-    this.isCompleting.set(false);
-  }
-
-  private newOrganizationNameToActorName(): void {
-    this.newOrganizationForm.controls.companyName.valueChanges
-      .pipe(takeUntilDestroyed())
-      .subscribe((value) => {
-        this.newActorForm.controls.name.setValue(value);
-      });
```

Comment:
question: this method made sure that the organization's name was also set as the market participant's name. Does that still happen or is it not necessary anymore?